src/scantailor/app/ui/widgets/color_pickup.py

```
@@ -0,0 +1,352 @@
+
+    Also supports color pickup from the image via ColorPickupInteraction.
+
+    Signals:
+        color_changed: Emitted when the color changes (QColor).
+    """
+
+    color_changed = Signal(QColor)
+
+    def __init__(
+        self,
+        initial_color: QColor = QColor(255, 255, 255),
```

Ruff (0.15.0)

[warning] Do not perform function call `QColor` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)